src/main/java/baritone/behavior/PathingBehavior.java

```diff
@@ -372,6 +377,16 @@ public CalculationContext secretInternalGetCalculationContext() {
         return context;
     }
 
+    public Optional<Double> estimatedTicksToGoal(){
+        if (goal == null){
+            return Optional.empty();
+        }
+        BetterBlockPos currentPos = ctx.playerFeet();
+        double current = goal.heuristic(currentPos.x, currentPos.y, currentPos.z);
+        double start = goal.heuristic(startPosition.x, startPosition.y, startPosition.z);
+        return Optional.of(current * ticksElapsedSoFar / (start - current));
+    }
```

Comment:
formatting

Reply:
Can you be more specific on what I should change please?

Reply:
```diff
+    public Optional<Double> estimatedTicksToGoal() {
-    public Optional<Double> estimatedTicksToGoal(){
+        if (goal == null) {
-        if (goal == null){
=            return Optional.empty();
=        }
+
=        BetterBlockPos currentPos = ctx.playerFeet();
=        double current = goal.heuristic(currentPos.x, currentPos.y, currentPos.z);
=        double start = goal.heuristic(startPosition.x, startPosition.y, startPosition.z);
=        return Optional.of(current * ticksElapsedSoFar / (start - current));
=    }
```

Reply:
Well if that's an issue I'll fix it ofc